studiopress/pattern-manager
# Filter registration of the `core/pattern` block

* This [filters](https://github.com/studiopress/pattern-manager/pull/137/files#diff-5b83ab67f2de162b1f57dbcfc70df34fd308abdbcad9a538070252ee497d55c5R22) registration of the `'core/pattern'` block, instead of [replacing it with PHP](https://github.com/studiopress/pattern-manager/pull/137/files#r1162122854). It should work the same as https://github.com/studiopress/pattern-manager/pull/129.

* Much of this PR's `<PatternEdit>` component is changed in https://github.com/studiopress/pattern-manager/pull/138, but that doesn't change the filtering.

### How to test
1. Create a new pattern
2. Add a 'PM Pattern' block: 

<img width="493" alt="Screenshot 2023-04-10 at 4 05 24 PM" src="https://user-images.githubusercontent.com/4063887/231008086-a7e02102-e6ef-4039-a81b-d322fb4fcb97.png">

3. Click a 'Pattern To Use': 

<img width="639" alt="Screenshot 2023-04-10 at 4 07 23 PM" src="https://user-images.githubusercontent.com/4063887/231008402-b3f753e5-475f-4d3b-997d-1fcef835cd68.png">


4. Expected: There's no error: 

<img width="929" alt="Screenshot 2023-04-10 at 4 08 15 PM" src="https://user-images.githubusercontent.com/4063887/231008352-a83a7185-5f2b-4fc4-88ef-4d7bc1474302.png">

(This was already working in https://github.com/studiopress/pattern-manager/pull/129)
